Magento_Theme/layout/default.xml

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0"?>
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
+        <!-- Remove -->
+        <referenceBlock name="copyright" remove="true" />
+        <referenceBlock name="report.bugs" remove="true" />
+
         <referenceBlock name="logo">
             <arguments>
                 <argument name="logo_width" xsi:type="number">185</argument>
             </arguments>
         </referenceBlock>
+        <referenceContainer name="footer">
+        <block class="Magento\Theme\Block\Html\Header\Logo" name="logo-footer" template="Magento_Theme::footer-logo.phtml" before="-" />
```

Comment:
you don't need to create a custom template. use the same as for header. just change something via XML arguments. Hardcode nothing to the HTML layout directly

Reply:
don't give name like logo-footer. it could be used not only there. but if this logo has difference in color, so, name it like `logo-white.svg`.
See wiki how to provide a new path SVG image via XML arguments